sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clientTelemetry/MetricInfo.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.clientTelemetry;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MetricInfo {
+    private String metricsName;
+    private String unitName;
+    private double mean;
+    private long count;
+    private double min;
+    private double max;
+    private Map<Double, Double> percentiles = new HashMap<>();//strict contract, ,  validation on key required
```

Comment:
is this accessed from multiple threads?

Reply:
removed the object creation from here. This is getting created in a single thread every 10 min in client telemetry, so we don't need concurrency.